### PR TITLE
[release] Update version in package.xml when running make release

### PR DESCRIPTION
**Pull request by wxmerkt**
Potential solution to #483

### CODE REVIEW

File: release.cmake [hunk 1]
```diff
@@ -65,6 +65,14 @@ MACRO(RELEASE_SETUP)
       && ! test x$$VERSION = x
         || (echo "Please set a version for this release" && false)
       && cd ${PROJECT_SOURCE_DIR}
+      # Update version in package.xml if it exists
+      && if [ -f "package.xml" ]; then
+          (echo "Updating package.xml to $$VERSION"
+           && sed -i \"s!<version>[0-9].[0-9].[0-9]</version>!<version>$$VERSION</version>!g\" package.xml
+           && ${GIT} add package.xml
+           && ${GIT} commit -m "Update package.xml version to $$VERSION"
```
Comment (jcarpent) [inline release.cmake:73]:
```suggestion
           && ${GIT} commit -m "release: Update package.xml version to $$VERSION"
```

**Review by jcarpent — APPROVED** [review]
Thanks a lot @wxmerkt. This looks nice to me.
I made a simple suggestion ;)

**Review by nim65s — APPROVED** [review]
I guess the assumption that the `version` tag is on a single line is good enough :)

File: release.cmake [hunk 2]
```diff
@@ -65,6 +65,14 @@ MACRO(RELEASE_SETUP)
       && ! test x$$VERSION = x
         || (echo "Please set a version for this release" && false)
       && cd ${PROJECT_SOURCE_DIR}
+      # Update version in package.xml if it exists
+      && if [ -f "package.xml" ]; then
+          (echo "Updating package.xml to $$VERSION"
+           && sed -i \"s!<version>[0-9].[0-9].[0-9]</version>!<version>$$VERSION</version>!g\" package.xml
```
Comment (jmirabel) [inline release.cmake:71]: Don't you need to add some `+` to the regex to support version `10.234.15423542` ?

Comment (wxmerkt) [reply release.cmake:71]: Great catch, thank you. I just added a commit to fix it

**Review by jmirabel — APPROVED** [review]

**Pull request merged**